mne/utils/docs.py

```diff
@@ -1860,8 +1860,6 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
     channel positions from the montage. Valid strings are the names of the
     built-in montages that ship with MNE-Python; you can list those via
     :func:`mne.channels.get_builtin_montages`.
-    You may also want to consult the documentation of
-    :class:`mne.channels.DigMontage` for additional information.
```

Comment:
I don't think this is helpful at all, as basically one of the first things the DigMontage docs says is, "don't do this at home, ask an expert function to do it for you!" essentially sending you running around in circles. So I dropped it.